File: web/web/src/models/ProduktDto.ts
export interface ProduktDto {
    id : number;
    nazev : string;
    popis : string;
    kategorieId : number;
    cena : number;
    nakupniCena : number;
    minimalniSkladovaneMnozstvi : number;
    ean : string;
    hmotnost : number;
}

File: application/src/main/java/com/kodat/skladovysystem/Dto/ProduktDto.java
package com.kodat.skladovysystem.Dto;

import jakarta.validation.constraints.PositiveOrZero;

import java.math.BigDecimal;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class ProduktDto {
    private long id;

    private String nazev;

    private String popis;

    private Long kategorieId;

    private BigDecimal cena;

    private BigDecimal nakupniCena;

    @PositiveOrZero
    private int minimalniSkladovaneMnozstvi;

    private String ean;

    @PositiveOrZero
    private double hmotnost;
}
